mail_footer_notified_partners/models/mail_followers.py

```diff
@@ -57,7 +57,7 @@ def _notify(
             ctx.update({
                 'partners_to_notify': partners_to_notify,
             })
-        return super(MailNotification, self)._notify(
+        return super(MailNotification, self.with_context(ctx))._notify(
             message_id,
             partners_to_notify=partners_to_notify,
             force_send=force_send, user_signature=user_signature, context=ctx)
```

Comment:
Without that context we get back to where we were ... the notified partners aren't added/returned. That's actually the only change I made.